index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="./css/styles.css">
+    <script src="https://kit.fontawesome.com/cebf050324.js" crossorigin="anonymous"></script>
+  </head>
+  <body>
+    <div class="background">
+      <nav>
+        <ul class="list-format font-color">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
```

Comment:
- Let's wrap the nav in the header as it's semantically the container used to wrap the introductory content of a webpage to improve your semantic usage.